extensions/amp-analytics/0.1/visibility-helper.js

```diff
@@ -176,26 +206,26 @@ export class VisibilityHelper {
   }
 
   /**
-   * Sets whether this object is blocked. Blocking means that visibility is
-   * not ready to be calculated, e.g. because an element has not yet
+   * Sets whether this object is ready. Ready means that visibility is
+   * ready to be calculated, e.g. because an element has been
    * sufficiently rendered. See `getVisibility()` for the final
    * visibility calculations.
-   * @param {boolean} blocked
+   * @param {boolean} ready
    */
-  setBlocked(blocked) {
-    this.blocked_ = blocked;
+  setReady(ready) {
+    this.ready_ = ready;
     this.update();
   }
 
   /**
    * Returns the final visibility. It depends on the following factors:
    *  1. This object's visibility.
-   *  2. Whether the object is blocked.
+   *  2. Whether the object is ready.
    *  3. The parent's visibility.
    * @return {number}
    */
   getVisibility() {
```

Comment:
It can be by current dependencies. But it's somewhat of a clear API. Do you think it should be?